Manifests/ManagedPreferencesApplications/com.github.macadmins.Nudge.plist

```diff
@@ -996,6 +1001,22 @@ Specify one array to enforce a single Operating System version across all machin
 			<key>pfm_type</key>
 			<string>dictionary</string>
 		</dict>
+		<dict>
+		    <key>pfm_description</key>
+			<string>When configuring Nudge using a JSON-formatted preferences file located outside of the default location, set this key to the custom location using either a remote (https://) or a local (file://) URL, and leave other keys in this payload unset. Leave this key unset when configuring Nudge using the other keys in this payload.</string>
+			<key>pfm_documentation_url</key>
+			<string>https://github.com/macadmins/nudge/wiki#setting-the-json-url-in-mobileconfig-profile</string>
+			<key>pfm_format</key>
+			<string>^(https?://.+|file://.+)$</string>
```

Comment:
Spacing got a little funky here, think this whole section is off because 4 spaces are used instead of tabs.  I don't care either way so long as whatever is done is consistent throughout